spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/GeneratedTestClassMother.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.contract.verifier.builder;
+
+/**
+ * @author Tim Ysewyn
+ */
+public class GeneratedTestClassMother {
```

Comment:
Maybe `testClassGenerator` or `GeneratedTestClassProvider` rather than  `GeneratedTestClassMother`?

Reply:
Renamed to `GeneratedTestClassProvider`